### ISSUE TITLE
Hydra's head won't load

### ISSUE THREAD

**Issue by DaFrancc** [TeamTwilight/twilightforest#1023]
Forge version: 1.16.5-forge-36.0.48
Twilight Forest version: 4.0.374-universal
Link to crash log: No crash

Steps to reproduce:
1. Unsure

What I expected to happen: I was going to fight the Hydra

What happened instead: When I got there, the body was loaded, but the heads were not loaded and they were shooting fireballs at me.

I have optifine installed with no shaders.

**Comment by GizmoTheMoonPig** [comment]
If you remove optifine, do you still have the same issue?

**Comment by DaFrancc** [comment]
> If you remove optifine, do you still have the same issue?

I was playing with a custom Vanilla+++ modpack so I'm not 100% sure it can all be blamed on Optifine (I also have Immersive Portals). I tested it anyways, I removed all mods except Twilight Forest, made a brand new world, and the Hydra loads. However, when I relaunched the game with Optifine and loaded that same world, it crashes. Link to the crash report here: [https://gist.github.com/DaFrancc/eb1dd3da40ff5cbb4492a6eebeb63f3d](url)
